types/react-native-web/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "private": true,
+    "name": "@types/react-native-web",
+    "version": "0.0.9999",
```

Comment:
Not sure what version should I use for this 🤔 If you are a DefinietlyTyped maintainer, I'm open for your input!

Reply:
@weswigham I would appreciate your input here 🙇

Reply:
@sheetalkamat Was 0.0.9999 the right choice here?